src/views/FindOrder.vue

```diff
@@ -254,6 +254,13 @@ export default defineComponent ({
     }
   },
   methods: {
+    getTotalItems(orders: any){
```

Comment:
We can remove this method as it's unused.

Reply:
Removed